modules/etcd-defrag.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="etcd-defrag_{context}"]
+= Defragmenting etcd data
+
+Manual defragmentation must be performed periodically to reclaim disk space after etcd history compaction and other events cause disk fragmentation.
+
+History compaction is performed automatically every five minutes and leaves gaps in the back-end database. This fragmented space is available for use by etcd, but is not available to the host file system. You must defragment etcd to make this space available to the host file system.
+
+// TODO: Should we add a warning like, if you do not defragment etcd, you might run out of disk space ... and about the cluster getting put into the read/delete mode only?
```

Comment:
@retroflexer Do you think that it would be worthwhile to include a warning on this? I read in the etcd docs about the system getting put into read/delete only mode, so I wasn't sure if that truly could happen, and if we needed to give a warning on it?